packages/govern-subgraph/schema.graphql

```diff
@@ -67,7 +67,7 @@ type ContainerEvent @entity {
   id: ID!
   type: ContainerEventType
   createdAt: BigInt
-  data: [String]!
+  data: [String!]!
```

Comment:
Btw, small thing, let's change this to bytes so we can encode anything here!

Reply:
The main reason I didn't do it yet was because I'm not sure hot to proper handle `BigInt`. How do you suggest we encode them?